microsoftgraph/microsoft-graph-explorer-v2
# Requests incremental permissions consent

## Overview

Fetches required scopes for a sample query from the Permissions API when user clicks on `Run Query` and requests permissions consent incrementally from least privilege to most using MSAL.

### Notes

This PR closes #172 


## Testing Instructions

* Log on to the tst environment.
* Click on a sample query that requires scopes not yet consented to.
* Notice a dialog box pop up with the required scope for the requested sample query for consent.
